lib/Mojolicious/resources/templates/mojo/debug.html.ep

```diff
@@ -170,24 +170,30 @@
                   <script>
                     let current = '#context';
                     function mojoShowcase() {
-                      $('#showcase').on('click', function () {
-                        $(this).unbind('click');
-                        $(current).slideToggle('slow', function () {
-                          current = current == '#context' ? '#insight' : '#context';
-                          $(current).slideToggle('slow', function () {
-                            mojoShowcase();
-                          });
-                        });
-                      });
+                      const showcase = document.querySelector('#showcase');
+                      function onClickShowcase() {
+                        showcase.removeEventListener('click', onClickShowcase);
+                        let currentEl = document.querySelector(current);
+                        currentEl.classList.toggle('d-none');
+                        current = current == '#context' ? '#insight' : '#context';
+                        currentEl = document.querySelector(current);
+                        currentEl.classList.toggle('d-none');
+                        mojoShowcase();
+                      };
+                      showcase.addEventListener('click', onClickShowcase);
                     }
                     mojoShowcase();
-                    $('#insight').toggle();
+                    function toggleInsight() {
+                      const insightEl = document.querySelector('#insight');
+                      insightEl.classList.toggle('d-none');
+                    }
+                    toggleInsight();
```

Comment:
Why is this a function?

Reply:
Because I was returning if I had an error, it is no longer a function now, sorry.